pkg/splunk/enterprise/configuration.go

```diff
@@ -479,7 +486,7 @@ func getSplunkStatefulSet(ctx context.Context, client splcommon.ControllerClient
 func getSmartstoreConfigMap(ctx context.Context, client splcommon.ControllerClient, cr splcommon.MetaObject, instanceType InstanceType) *corev1.ConfigMap {
 	var configMap *corev1.ConfigMap
 
-	if instanceType == SplunkStandalone || instanceType == SplunkClusterManager {
+	if instanceType == SplunkStandalone || instanceType == SplunkClusterManager || instanceType == SplunkClusterMaster {
```

Comment:
Maybe just wrap this `instanceType == SplunkClusterManager || instanceType == SplunkClusterMaster` in a function and use that function everywhere, might be easy to track it and in future we only need to make change at only one place.

Reply:
Done